### PR TITLE
refactor(StakeManager): make function names more descriptive

### PR DESCRIPTION
**Pull request by 0x-r4bbit**
Some of the functions on our contracts were confusing. This commit changes them so they describe what they actually do.

## Checklist

Ensure you completed **all of the steps** below before submitting your pull request:

- [x] Added natspec comments?
- [ ] Ran `forge snapshot`?
- [ ] Ran `pnpm gas-report`?
- [x] Ran `pnpm lint`?
- [x] Ran `forge test`?
- [x] Ran `pnpm verify`?

### CODE REVIEW

File: contracts/StakeManager.sol [hunk 1]
```diff
@@ -66,7 +66,7 @@ contract StakeManager is Ownable {
         _;
     }
 
-    modifier onlyInitialized(address account) {
+    modifier onlyAccountInitialized(address account) {
```
Comment (0x-r4bbit) [inline contracts/StakeManager.sol:69]: I've changed this one as `onlyInitialized` is commonly used in OZ `Initializable` contracts

File: contracts/StakeManager.sol [hunk 2]
```diff
@@ -106,7 +106,7 @@ contract StakeManager is Ownable {
     /**
      * @notice Process epoch if it has ended
      */
-    modifier processEpoch() {
+    modifier finalizeEpoch() {
```
Comment (0x-r4bbit) [inline contracts/StakeManager.sol:109]: Really wat this modifier does is finalizing the epoch if it's duration has passed.
There's an argument to be made that there's cases where it won't finalize, but I still think it's better than "process epoch" which can mean many things.

File: contracts/StakeManager.sol [hunk 3]
```diff
@@ -324,10 +348,10 @@ contract StakeManager is Ownable {
     /**
      * @dev Only callable from old manager.
      * @notice Increase total MP from old manager
-     * @param _increasedMP amount MP increased on account after migration initialized
+     * @param _amount amount MP increased on account after migration initialized
      */
-    function increaseMPFromMigration(uint256 _increasedMP) external onlyOldManager {
-        totalSupplyMP += _increasedMP;
+    function increaseTotalMP(uint256 _amount) external onlyPreviousManager {
```
Comment (0x-r4bbit) [inline contracts/StakeManager.sol:353]: The entire "increasedMP" notion in this contract felt a bit off to me. 
In this case here, really what we're doing is adjusting the total MP of the next manager.

File: contracts/StakeManager.sol [hunk 4]
```diff
@@ -437,23 +458,23 @@ contract StakeManager is Ownable {
...
     /**
-     * @notice Calculates increased multiplier points for given balance and time
+     * @notice Calculates multiplier points to mint for given balance and time
      * @param _balance balance of account
      * @param _deltaTime time difference
-     * @return _increasedMP increased multiplier points
+     * @return multiplier points to mint
      */
-    function _getIncreasedMP(uint256 _balance, uint256 _deltaTime) private pure returns (uint256 _increasedMP) {
+    function _getMPToMint(uint256 _balance, uint256 _deltaTime) private pure returns (uint256) {
```
Comment (0x-r4bbit) [inline contracts/StakeManager.sol:477]: Also here, this function really just calculates the MP that should be minted

File: test/StakeManager.t.sol [hunk 5]
```diff
@@ -629,27 +628,6 @@ contract ExecuteAccountTest is StakeManagerTest {
...
-        console.log("rewardAddress :", rewardAddress);
-        console.log("##### balance :", balance);
-        console.log("### initialMP :", initialMP);
-        console.log("### currentMP :", currentMP);
-        console.log("#### lastMint :", lastMint);
-        console.log("### lockUntil :", lockUntil);
-        console.log("####### epoch :", epoch);
-        console.log("===============");
-    }
-
```
Comment (0x-r4bbit) [inline test/StakeManager.t.sol:652]: This entire function was not used anywhere.

**Review by 3esmit — APPROVED** [review]
Makes sense.

**Pull request merged**